src/main/java/com/aws/greengrass/clientdevices/auth/ClientDevicesAuthService.java

```diff
@@ -68,6 +70,8 @@
 @ImplementsService(name = ClientDevicesAuthService.CLIENT_DEVICES_AUTH_SERVICE_NAME)
 public class ClientDevicesAuthService extends PluginService {
     public static final String CLIENT_DEVICES_AUTH_SERVICE_NAME = "aws.greengrass.clientdevices.Auth";
+    private static final ObjectMapper MAPPER = SerializerFactory.getRecipeSerializerJson();
```

Comment:
I think you want `getConfigurationSerializerJson` to avoid changing capitalization

Reply:
We shouldn't fail on unknown though. That prevents forward compatibility

Reply:
yeah, ok if we're fine with that behavior change then yeah can add it in

Reply:
done